Classes/conditional.py

```diff
@@ -1,59 +1,114 @@
 class Conditional:
     """
     Conditionals are used in Rooms, as anything that may change status.
-    It takes the conditionalDescription JSON data from the
+    It takes the "conditions" JSON data from the
     Room where it is located.
 
-    Data parameter is a dict of conditional descriptions from Room.
+    Input "name" as String for this condition.
+    Input "loop" expects a string reading "True" or "False".
+    Input "descriptions" takes a list of descriptions per step.
     """
-    def __init__(self, name, status, trueDesc, falseDesc):
-        # Name the conditional
-        self.name = name.lower()
+    def __init__(self, names, loop, descriptions):
+        # Names contains a sequence of all trigger words for this condition
+        self.names = names
 
-        # Get its current (or default) status
-        statusString = status
-        self.status = True
-        if statusString.lower() == "false":
-            self.status = False
+        # Attribute "trigger" is the current name of this condition,
+        # which is required to proceed to the next step.
+        self.trigger = self.names[0]
 
-        # Get both possible descriptions
-        self.trueDesc = trueDesc
-        self.falseDesc = falseDesc
+        # Loop attribute determines if this is a condition that can
+        # toggle back and forth (True) or if it only progresses forward (False)
+        if loop.lower() == "true" or loop is True:

```

Comment:
Do we need such a complex "if" condition? Won't "loop" either be "True" or "False"?

Reply:
I wrote it this way because it covers all cases where the JSON file might be read as "true" or "True" or True (the boolean value). While it can be simplified and still work, it would not be as robust. Since they are both still just one line of code, I opted for robustness. But I'm open to changing it, if needed.